rewrite-java/src/main/java/org/openrewrite/java/JavaVisitor.java

```diff
@@ -920,6 +916,14 @@ public J visitMethodInvocation(J.MethodInvocation method, P p) {
         return m;
     }
 
+    public J visitModifier(J.Modifier modifer, P p) {
```

Comment:
I haven't seen the corresponding override of acceptJava() on J.Modifier. is that file missing here?

Reply:
Added. Now failing tests passing too...